server/src/typeDefs/query.ts

```diff
@@ -2,9 +2,13 @@ import { gql } from 'apollo-server'
 
 export const query = gql`
   type Query {
-    usersPagination(page: Float!, where: UserWhereInput): UsersPagination!
-    user(userId: String!): User!
+    user: User!
```

Comment:
loggedInUser or me. `user` is basically the worst option here as its really ambiguous